### PR TITLE
Added the HTTP server document

### LINKED ISSUE #336: Added the HTTP server document.
### What would you like to be added?

Added the HTTP server document.

### Why is this needed?

The HTTP server capability is added to the Sermant 2.0.0 version.

### PR DESCRIPTION
**Pull request by zwmagic**
## What type of PR is this?

/Doc

## What this PR does / why we need it?

Added the HTTP server document

## Which issue(s) this PR fixes:

Fixes #336

### CODE REVIEW

File: docs/zh/document/developer-guide/sermant-httpserver-service.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# HTTP Server服务
```
Comment (lilai23) [inline docs/zh/document/developer-guide/sermant-httpserver-service.md:1]: Add this doc under title: "开发者指南" in config.js

File: docs/zh/document/developer-guide/sermant-httpserver-service.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+本文介绍如何在 Sermant 中实现HTTP接口服务的开发。
+
+## 功能介绍
+Sermant 提供了内置的 HTTP Server 功能，允许开发者在插件中通过简单的配置和实现相关接口，快速搭建起可以对外提供 HTTP 接口的服务。此功能使得外部系统可以访问这些接口，实现数据的交互和处理。HTTP Server 支持常见的 HTTP 方法，如 GET、POST、PUT、DELETE 等，使得开发者能够根据不同的业务需求，灵活地设计和实现功能。
+
+## 开发示例
+要使用 Sermant 的 HTTP Server 功能，首先需要实现 `HttpRouteHandler` 接口，并在实现类上添加 `@HttpRouteMapping` 注解来指定路由信息。以下是一个具体的示例：
```
Comment (lilai23) [inline docs/zh/document/developer-guide/sermant-httpserver-service.md]: 首先应该开启HTTP Server的开关，详细配置可参考[配置章节](#配置)

File: docs/zh/document/developer-guide/sermant-httpserver-service.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# HTTP Server服务
+
+本文介绍如何在 Sermant 中实现HTTP接口服务的开发。
+
+## 功能介绍
+Sermant 提供了内置的 HTTP Server 功能，允许开发者在插件中通过简单的配置和实现相关接口，快速搭建起可以对外提供 HTTP 接口的服务。此功能使得外部系统可以访问这些接口，实现数据的交互和处理。HTTP Server 支持常见的 HTTP 方法，如 GET、POST、PUT、DELETE 等，使得开发者能够根据不同的业务需求，灵活地设计和实现功能。
```
Comment (lilai23) [inline docs/zh/document/developer-guide/sermant-httpserver-service.md]: 建议举个使用该功能的例子

File: docs/zh/document/developer-guide/sermant-httpserver-service.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+# 使用 JDK 内置的 Http Server
+httpserver.type=simple
+# 监听端口
+httpserver.port=8080
+# 线程池核心线程数
+httpserver.serverCorePoolSize=10
+# 线程池最大线程数
+httpserver.serverMaxPoolSize=20
+```
+通过上述配置和开发示例，开发者可以快速搭建起一个 HTTP Server 服务，对外提供 HTTP 接口。
```
Comment (lilai23) [inline docs/zh/document/developer-guide/sermant-httpserver-service.md]: 这里少了agent.service.httpserver.enable开关的说明

File: docs/zh/document/developer-guide/sermant-httpserver-service.md (hunk 1 above)
Comment (zwmagic) [reply docs/zh/document/developer-guide/sermant-httpserver-service.md:1]: fixed

File: docs/zh/document/developer-guide/sermant-httpserver-service.md (hunk 2 above)
Comment (zwmagic) [reply docs/zh/document/developer-guide/sermant-httpserver-service.md]: fixed

File: docs/zh/document/developer-guide/sermant-httpserver-service.md (hunk 3 above)
Comment (zwmagic) [reply docs/zh/document/developer-guide/sermant-httpserver-service.md]: fixed

File: docs/zh/document/developer-guide/sermant-httpserver-service.md (hunk 4 above)
Comment (zwmagic) [reply docs/zh/document/developer-guide/sermant-httpserver-service.md]: fixed

**Review by lilai23 — APPROVED** [review]

**Pull request merged**